.github/workflows/ci-build-image.yml

```diff
@@ -10,7 +12,7 @@ on:
       build-arch:
         required: true
         type: string
-      skip-itests:
+      run-tests-only:
```

Comment:
we can get rid of this :)

Reply:
Im blind 😂

Reply:
I will celebrate if this PR is merged. I see all related code in my sleep too😂

Reply:
that's how I felt with /build_test haha